geth/txqueue/txqueue_manager_test.go

```diff
@@ -124,36 +155,36 @@ func (s *TxQueueTestSuite) TestCompleteTransactionMultipleTimes() {
 
 	txQueueManager.SetTransactionReturnHandler(func(queuedTx *common.QueuedTx, err error) {
 		s.Equal(tx.ID, queuedTx.ID)
-		s.Equal(errTxAssumedSent, err)
+		s.NoError(err)
 	})
 
 	err := txQueueManager.QueueTransaction(tx)
 	s.NoError(err)
 
 	var wg sync.WaitGroup
 	var mu sync.Mutex
-	completeTxErrors := make(map[error]int)
+	completeTxErrors := map[error]int{}
```

Comment:
I think it should be possible to get rid of this map and only assert `errCompleteTransaction` to be always `nil`.